helpers/middleman/menu.rb

```diff
@@ -0,0 +1,266 @@
+load File::expand_path('./locale.rb', __dir__)
+load File::expand_path('./page.rb', __dir__)
+
+module MiddlemanMenuHelpers
+  include MiddlemanLocaleHelpers # for current_locale_obj method
+  include MiddlemanPageHelpers # for locale_page method
+
+
+  def get_menu_config(locale_obj: current_locale_obj)
+    return [
+      get_features_menu(locale_obj: locale_obj),
+      get_benefits_menu(locale_obj: locale_obj),
+      get_pricing_menu(locale_obj: locale_obj),
+      get_resources_menu(locale_obj: locale_obj)
+    ]
+  end
+
+  private
+
+  def get_features_menu(locale_obj: current_locale_obj)
+    return {
+      label: 'Features',
+      rows: [
+        {
+          columns: [
+            {
+              label: 'Features',
+              links: [
+                {
+                  icon: 'trend-up',
+                  icon_hover: 'chart-line-up-bold',
+                  title: 'Performance',
+                  href: '/TODO',
```

Comment:
These TODOs are really for the content team and should be caught by QA blocking a deploy 😉